roles/bootstrap/tasks/main.yml

```diff
@@ -104,11 +105,24 @@
     name: "{{ bootstrap_default_common_packages }}"
     state: present
 
+- name: Check if reboot required
+  ansible.builtin.stat:
+    path: /var/run/reboot-required
+  register: reboot_required_file
+
 # Disable swap
 - name: Disable SWAP in fstab
   ansible.builtin.replace:
     path: /etc/fstab
     regexp: '^([^#].*?\sswap\s+sw\s+.*)$'
     replace: '# \1'
   when: bootstrap_swap_disable
-  notify: Disable SwAP
+  register: bootstrap_swap_disabled
```

Comment:
Nice catch, but instead of registering the variable, you should remove the when condition on the handler https://github.com/ethpandaops/ansible-collection-general/blob/master/roles/bootstrap/handlers/main.yml#L8 . The `notify` call will already trigger the handler, it doesn't need the when condition.